README.md

```diff
@@ -8,31 +8,31 @@
 ```
 
 ## [PasscodeLock](https://github.com/arg0NNY/DiscordPlugins/tree/master/PasscodeLock) ![Status][status-official-badge] ![Version][passcodelock-version-badge] ![Downloads][passcodelock-downloads-badge] ![Likes][passcodelock-likes-badge]
-Protect your Discord with 4-digit passcode.
+Protect your Discord with an up to 15-digit long passcode.
```

Comment:
Just "with a passcode", cause I don't want to change it every time passcode format changes